.github/workflows/test.yml

```diff
@@ -13,13 +13,16 @@ jobs:
     runs-on: ubuntu-22.04
     timeout-minutes: 20
     steps:
-      - uses: actions/setup-go@v3
-        with:
-          go-version: 1.19.x
       - uses: actions/checkout@v3.3.0
         with:
           path: src/github.com/containerd/nerdctl
           fetch-depth: 100
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.x
```

Comment:
can we expose the go version for the entire workflow, to have a single point change:
```
      - uses: actions/setup-go@v3
        with:
          go-version: ${{ env.GO_VERSION }}
          check-latest: ${{ env.CHECK_LATEST }}
          cache-dependency-path: ${{ env.CHECK_DEPENDENCY_PATH }}
```

Reply:
We do not need it as src/github.com/containerd/nerdctl is mapped at checkout actions in project checks only.